doc/qbk/2.0.quicklook.qbk

```diff
@@ -126,18 +113,27 @@ To get a decoded string, call the same function without the word `encoded`:
 ]]]
 
 The decoded functions return a __const_string__, which is a library type
-that models a read-only string. __const_string__ retains ownership of the
-underlying buffer and uses a type-erased allocator. An extra built-in
-buffer is also provided to avoid allocations for any string smaller than
-this [link url.ref.boost__urls__const_string.builtin_capacity `builtin_capacity`].
+that models a read-only string. This class retains ownership of the
+underlying buffer and uses a type-erased allocator.
+
+An extra built-in buffer is also provided to avoid allocations for any
```

Comment:
I don't think we should advertise this in the public interface. We can just say "const_string" is optimized for the use case of non-modifiable transformed strings, or say nothing.

Reply:
Rationale: if we say this publicly then if we decide to change it, then it would be a breaking change. Not as big as changing a function signature, but still - better not to advertise it.

Reply:
My rationale is that was already part of the API because we made the built-in capacity public. It was just to save some typing in the tests but still. Maybe we should make this private and use 32 directly in the tests.

Reply:
hmm yeah, we should make `builtin_capacity` private.